pkg/profileparser/profileparser_test.go

```diff
@@ -579,6 +582,9 @@ var _ = Describe("Testing parse rules", func() {
 
 		stdParser := newStandardParser()
 		nonce := names.SimpleNameGenerator.GenerateName("pb-")
+		zaplog, _ := zap.NewDevelopment()
+		log = zapr.NewLogger(zaplog)
```

Comment:
This is unused? What's it do?

Reply:
It sets the global log instance just in the test to Development. Thus printing the reconciler's log messages when running tests in verbose mode.

Reply:
At some point we should move away from those log instances in global variables and instantiate them as part of the reconcile object.

Reply:
Ah, sounds good.